### PR TITLE
chore: update ship.js config

### PR DESCRIPTION
**Pull request by benhinchley**
As per https://github.com/algolia/search-insights.js/pull/490#discussion_r1310172581

### CODE REVIEW

File: ship.config.js
Comment (benhinchley) [inline ship.config.js:13]: @dhayab Not sure if you want @algolia/frontend-experiences-web to stay on the release PRs or not? Let me know and I can add the team back here.

**Review by jonathaningram — APPROVED** [review]
LGTM. Leave it to you to decide to address your pending comment now or later.

File: ship.config.js
Comment (dhayab) [reply ship.config.js:13]: I’m good with that change, it’s in line with what’s in the CODEOWNERS file.

**Pull request merged**
